.github/workflows/ci.yml

```diff
@@ -0,0 +1,125 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pc2fromecs/jdk8hugo
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.x
```

Comment:
Is this the same version of Node that we were building with on GitLab?   I know that we've had some problems with Node compatibility between our local machines and what's on GitLab....

Reply:
we were pinning to node 14.